SchoolNormalization.md

```diff
@@ -41,8 +45,9 @@ Example: https://api.careerbuilder.com/core/normalizedschools?query=georgia%20te
 
 #Response Information
 
-The response returns a single data node which contains a list of normalized schools. These normalized schools are ordered by the confidence score. Each normalized shool has a normalized school name (string), a unique ID (string), a country (string), and a confidence (double). Confidence scores range from 0.0 to 2.0.
+The response returns a single data node which contains a list of normalized schools. These normalized schools are ordered by the confidence score. Each normalized shool has a normalized school name (string), a unique ID (string), a confidence (double), a country (string), a city (string), a state (string), an ipeds_id (string), and an ipeds_name (string). Confidence scores range from 0.0 to 2.0.
 
+Regarding the two fields ipeds_id and ipeds_name; ipeds stands for Integrated Postsecondary Education Data System. The unique Id and Name are assigned to every school registered with the National Center for Education Statistics. 
```

Comment:
I would use a "," instead of ";". pardon me if I am wrong.
Besides I prefer "unique id and name" instead of "unique Id and Name", or just use "unique ipeds_id and ipeds_name".